runtime/src/server/middleware/get_page_handler.ts

```diff
@@ -260,9 +260,15 @@ export function get_page_handler(
 				level0: {
 					props: preloaded[0]
 				},
-				level1: {
-					segment: segments[0],
-					props: {}
+				level1: error ? {
```

Comment:
I don't know if this should be here or not, as it assumes my changes/simplificates to create_app are present (which they don't seem to be).

Reply:
done